config/defaults.env

```diff
@@ -32,3 +32,7 @@ OTEL_EXPORTER_OTLP_ENDPOINT=https://collector.opensafely.org
 
 # WARNING: care needs to be taken if changing this. See playbook.md
 DATABASE_ACCESS_NETWORK=jobrunner-db
+
+# RAP Controller
+DJANGO_CONTROLLER_ALLOWED_HOSTS='*'
+DJANGO_DEBUG=False
```

Comment:
As this isn't namespaced to a component (ie it's not `DJANGO_CONTROLLER_DEBUG`), is it worth putting in its own section?

Reply:
Well it is specific to the controller Django app as that's the only thing that reads it. I think the problem is just that it isn't namespaced to the controller and it arguably should be.

Alternatively, it might be reasonable to say that this is only useful in development and therefore it should default to `False` and we don't need to set it at all. In general I prefer having to explicitly set config, but this is a slightly unusual case where the correct production setting is so unambiguous.

Reply:
It is used by Airlock too (hence no _CONTROLLER_ bit)

Reply:
The agent also needs `CONTROLLER_TASK_API_ENDPOINT` (unless we're planning to add this in the local.env only)

Reply:
Yeah, I deliberately put that in `docker-compose.yaml` for the time being because it's specific to Compose's networking config. When we move the Controller to a separate server then the config should go with the rest of the env settings as you say.

Reply:
On sharing the setting with Airlock, that feels a bit icky to me. I think either we need to namespace them both or, specifically for the DEBUG setting, we give it a default value. I'm leaning towards the latter here as I can't think of any reason we'd want to enable it in production.

Reply:
I didn't namespace it because all these CONTROLLER settings were only supposed to be temporary; when they're outside the backend, they won't need the extra CONTROLLER tag.

Reply:
Ah yeah, true. Well maybe we just leave it all as it is for now then.